applications/decentralized_well-being_game_api.md

```diff
@@ -0,0 +1,143 @@
+# Project Overview
+
+## Project Name
+
+Decentralized Well-being Game API
+
+## Team Name
+
+Health Hero
+
+## Payment Address
+
+0xc7ad868f9b421dabf7cdaa0e6c508ff74e2f1cfd
```

Comment:
Could you add which currency you would like to be paid in (BTC, USDT, DAI or kUSD)?

Reply:
@semuelle  USDT thanks

Reply:
Into the document, please.

Reply:
updated thanks @semuelle